.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Integration tests
+on:
+  pull_request:
+  # run them regularly, to catch changes in operating systems
+  schedule:
+  - cron: 0 4 * * MON
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Install Ansible
+        run: sudo apt install -y ansible
+
+      - name: Run test playbook
+        run: |
+          cd tests
+          sudo ansible-playbook --connection=local --inventory localhost, tests_default.yml
```

Comment:
... I added that to PR #34, where it makes more sense IMHO.

Reply:
> neither has "cleanup::" in it

only a few roles do this correctly - but since you are going to implement something that requires a cleanup section, you might as well do it the right way

Reply:
Resolving -- this is done in #34.